concepts/05 Widgets/Form/01 Getting Started with Form/40 Organize the Form Items/40 Add an Empty Space.md

```diff
@@ -0,0 +1,158 @@
+If you need to add an empty space between neighboring items, use an empty item. To create it, assign "empty" to the [itemType](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/EmptyItem/#itemType) property. To define how many columns the empty item must span, specify the [colSpan](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/EmptyItem/#colSpan) option. For the full list of available properties, visit the [Empty Item](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/EmptyItem/) section.
+
+In the following example, the empty item [spans](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/EmptyItem/#colSpan) two columns:
```

Comment:
Will be a screenshot here?
It's not easy to imagine the resulting 'pixels' from this declaration and why/when I need an empty item.